wiki/History_of_osu!/2018/en.md

```diff
@@ -10,13 +10,13 @@ After a six-month long hiatus, flyte posted on the [osu!next](https://osunext.tu
 
 The osu!mania 7K World Cup 2018 registrations opened. Community Choice 2017 (previously known as the "Best of" series) voting was opened.
 
-In osu!lazer, background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
+In osu!(lazer), background blurring and the hidden mod were added. osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hitsounds was added. osu!catch was vastly improved and autoplay was fixed to be more human-like.
 
 In osu!web, beatmap pack listings were improved, hype and nomination counters were visible to the beatmap listings. Obtaining a permalink to beatmap discussion post was made easier to copy, localisations can now resize the menu upon hovering over the locales, star ratings can be seen on the beatmap listing via hovering the difficulty icon, and sorting of beatmap discussions was added. A limit to only load 250 forum threads and unload others was implemented to avoid heavy memory consumption.
 
 peppy tweeted that the new t-shirt inventory (along with their new designs) has arrived. Six days later, they were up on the osu!store.
 
-The ability to see star ratings in the osu!direct was backported from osu!lazer. Lastly, thanks to nekodex, the scoreboard redesign was implemented.
+The ability to see star ratings in the osu!direct was backported from osu!(lazer). Lastly, thanks to nekodex, the scoreboard redesign was implemented.
```

Comment:
```suggestion
The ability to see star ratings in the osu!direct was backported from the lazer version of the osu! client. Lastly, thanks to nekodex, the scoreboard redesign was implemented.
```

Reply:
Not 100% sure which of these reads better (strictly speaking "lazer" is not a version but a release stream), but let's go with what you have for now.